sql/moz-fx-data-shared-prod/telemetry_dev_cycle_derived/glean_metrics_external_v1/query.py

```diff
@@ -0,0 +1,122 @@
+"""Glean metric data - download from API, clean and upload to BigQuery."""
+
+import logging
+
+import click
+import requests
+from google.api_core.exceptions import BadRequest
+from google.cloud import bigquery
+
+API_BASE_URL = "https://probeinfo.telemetry.mozilla.org"
+SCHEMA = [
+    bigquery.SchemaField("glean_app", "STRING"),
+    bigquery.SchemaField("metric", "STRING"),
+    bigquery.SchemaField("type", "STRING"),
+    bigquery.SchemaField("first_seen_date", "DATE"),
+    bigquery.SchemaField("last_seen_date", "DATE"),
+    bigquery.SchemaField("expires", "STRING"),
+]
+PROJECT_ID = "moz-fx-data-shared-prod"
+DATASET_ID = "telemetry_dev_cycle_derived"
+TABLE_NAME = "glean_metrics_external_v1"
+
+
+def get_api_response(url):
+    """Get json of response if the requests to the API was successful."""
+    response = requests.get(url)
+
+    if response.status_code != 200:
```

Comment:
Could replase this with `response.raise_for_status()`. This would raise an exception for error status code.